MessageCenter3/Models/Message/MessageRepository.cs

```diff
@@ -88,49 +84,48 @@ private void AddNewMessageReipient(int messageId, int recipientId, RecipientType
             }   
         }
 
-        public List<MessageOutputModel> GetMessagesFromTo(int senderId, int recipientId, RecipientType recipientType)
+        public List<MessageOutputModel> GetDialogue(UserMessageRequestModel request)
```

Comment:
The request model can't be used for the repository. It can be DTO, RequestModel usually is related to the Web API layer.

API -> Request/Response Model
BLL ->   DTO/DTO
DAL -> DTO/Entity model

Reply:
Got it. Will add DTO specific models.